apps/nbs-gateway/src/main/java/gov/cdc/nbs/gateway/landing/LandingServiceProvider.java

```diff
@@ -0,0 +1,36 @@
+package gov.cdc.nbs.gateway.landing;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.gateway.route.RouteLocator;
+import org.springframework.cloud.gateway.route.builder.RouteLocatorBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import gov.cdc.nbs.gateway.ui.UIService;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+@Configuration
+class LandingServiceProvider {
+
+  @Bean
+  LandingService landingService(
+      @Value("${nbs.gateway.landing.uri}") final String host,
+      @Value("${nbs.gateway.landing.base}") final String base) throws URISyntaxException {
+    URI uri = new URI(host);
+
+    return new LandingService(uri, base);
+  }
+
+  @Bean
+  RouteLocator landingRouteLocator(
+      final RouteLocatorBuilder builder,
+      @Value("${nbs.gateway.landing.uri}") final String host,
+      @Value("${nbs.gateway.landing.base}") final String base,
+      final UIService uiService) {
```

Comment:
suggestion (non-blocking):  Using the `UIService` here can be confusing.  The `base` of the `UIService` is blank so adding the path from `nbs.gateway.landing.base` is essentially the same thing as `LandingService.base()`.  It may be more clear that LandingService is being called here if it was an argument instead of `host`, `base`, and `uiService`